keyboards/an_achronism/tetromino/config.h

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2022 an_achronism (@an-achronism)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+// #define WS2812_PIO_USE_PIO1 // This appears to be optional and not required
+#define RGB_DI_PIN GP26 // Use GPIO26 (pin 31) for LED matrix logic
+#define DRIVER_LED_TOTAL 70
+// #define RGBLED_NUM 70 // Moved to info.json, under rgblight
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 255
+#define RGB_MATRIX_STARTUP_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
```

Comment:
```suggestion
#define DRIVER_LED_TOTAL 70
```

`// #define WS2812_PIO_USE_PIO1` should be removed.

`#define RGB_DI_PIN GP26` can be defined in info.json, added suggestion below.

`// #define RGBLED_NUM 70` is in info.json no point in having it commented here.

`#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 255` & `#define RGB_MATRIX_STARTUP_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS` are defaults and can be removed, this also makes it simpler if a user wants to change these settings in their keymap directory.